validation/superchain-version_test.go

```diff
@@ -27,16 +27,13 @@ func testContractsMatchATag(t *testing.T, chain *ChainConfig) {
 
 	client, err := ethclient.Dial(rpcEndpoint)
 	require.NoErrorf(t, err, "could not dial rpc endpoint %s", rpcEndpoint)
-	require.NotNil(t, chain.ContractsVersionTag, "Chain does not declare a contracts_version_tag")
 
-	isFaultProofs := *chain.ContractsVersionTag == "op-contracts/v1.4.0"
+	isFaultProofs := true
 
 	versions, err := getContractVersionsFromChain(*Addresses[chain.ChainID], client, isFaultProofs)
```

Comment:
If FPs are assumed to always be true going forward, why don't we just remove the `isFaultProofs bool` param from `getContractVersionsFromChain` and the corresponding codepath?

Reply:
Agreed, did that here [41567d6](https://github.com/ethereum-optimism/superchain-registry/pull/383/commits/41567d6e5c30b58c8b13f6e35e4bb5d763aad826)